src/redux/initialState.js

```diff
@@ -976,8 +976,39 @@ const initialState = {
   compare: {
     products: [],
   },
+  promotedProducts: {
+    firstProduct: {
+      title: 'HOT DEALS',
+      id: 'aenean-ru-bristique-4',
+      name: 'Aenean Ru Bristique 4',
+      category: 'bed',
+      price: 30,
+      oldPrice: '$40',
+      stars: 2,
+      promo: 'sale',
+      image:
+        'https://assets.pbimgs.com/pbimgs/ab/images/dp/wcm/202022/0016/claremont-bed-c.jpg',
+      newFurniture: true,
+      heart: true,
+      addCompare: true,
+    },
+    secondProduct: {
```

Comment:
Nazewnictow do zmiany - to nie jest drugi produkt tylko lista wyróżnionych produktów

Reply:
Poprawiłam błędy ale mam problem bo wszystko działa ale nie przechodzą mi testy.

Reply:
Już poprawiłam. Wszystko powinno działać